service_user.go

```diff
@@ -78,6 +86,30 @@ func (h *ServiceUsersHandler) Get(project, serviceName, username string) (*Servi
 	return nil, err
 }
 
+// Update modifies the given Service User in Aiven.
+func (h *ServiceUsersHandler) Update(project, service, username string, update ModifyServiceUserRequest) (*ServiceUser, error) {
```

Comment:
LGTM; though I would rather and an extra empty line after an `if` blocks for better readability, but this is just me. There are no complaints from `go fmt`

Reply:
You will be looking at it, so how can I say no? ;)